proto/buf/alpha/webhook/v1alpha1/event.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.webhook.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/repository.proto";
+import "buf/alpha/registry/v1alpha1/repository_commit.proto";
+import "buf/alpha/registry/v1alpha1/webhook.proto";
+import "google/protobuf/timestamp.proto";
+
+// EventService is the service which receives webhook events from Buf.
+service EventService {
```

Comment:
I went for [no-stutter](https://sben.dev/stutter/) when naming things in this proto, so we wouldn't have `webhook.WebhookEventService`, but `webhook.EventService` in the generated go code.

But, this isn't go code but proto code. Is it fine like this? or should I include the `Webhook` prefix everywhere?

Reply:
I am a fan of removing the redundancies in naming

Reply:
The registry services also do not start with `...registry.v1alpha1.Registry...Service/`, so this seems good to me too!